parsers/filter/README.md

```diff
@@ -0,0 +1,124 @@
+# Filter
+
+## Description
+
+Find elements where its key value matches the regex.
+
+Learn more about how to configure Specify in the API documentation: [https://specifyapp.com/developers/cli](https://specifyapp.com/developers/cli)
+
+## Interface
+
+```ts
+interface parser {
+  name: 'filter';
+  options: {
```

Comment:
Options type should match the OptionsType describe in the parser.
Type `string` type which is available for the regex option missing here.